core/src/main/java/tc/oc/pgm/score/ScoreMatchModule.java

```diff
@@ -81,14 +86,19 @@ public boolean hasMercyRule() {
 
   public int getScoreLimit() {
     checkState(hasScoreLimit());
-
     if (hasMercyRule()) {
       return this.mercyRule.getScoreLimit();
     }
-
+    if (hasScoreLimitOverride()) {
```

Comment:
You likely want to check this before you check mercy rules. Otherwise you won't be able to set an override if there's mercy rules defined in the map, as those will define the limit first